File: RQ2_Prompt_Elements/OpenAI_Data/HumanEvalPython_output/scenario1/id_71T2000Test.py
# id_71_Test.java
from scenario1.id_71 import triangle_area

import pytest

class id_71_Test:
    """
    Test class of id_71.py.
    It contains ten unit test cases for the :func:`scenario1.id_71.triangle_area`.
    """


    def test_triangle_area_1(self):
        """
        Test case for the :func:`scenario1.id_71.triangle_area`.
        """
        assert triangle_area(3, 4, 5) == 6.0
		
    def test_triangle_area_2(self):
        """
        Test case for the :func:`scenario1.id_71.triangle_area`.
        """
        assert triangle_area(13, 14, 15) == 84.0
		
    def test_triangle_area_3(self):
        """
        Test case for the :func:`scenario1.id_71.triangle_area`.
        """
        assert triangle_area(1, 1, 1) == 0.43
		
    def test_triangle_area_4(self):
        """
        Test case for the :func:`scenario1.id_71.triangle_area`.
        """
        assert triangle_area(1, 2, 3) == -1
		
    def test_triangle_area_5(self):
        """
        Test case for the :func:`scenario1.id_71.triangle_area`.
        """
        assert triangle_area(1, 1, 2) == -1
		
    def test_triangle_area_6(self):
        """
        Test case for the :func:`scenario1.id_71.triangle_area`.
        """
        assert triangle_area(1, 2, 2) == -1
		
    def test_triangle_area_7(self):
        """
        Test case for the :func:`scenario1.id_71.triangle_area`.
        """
        assert triangle_area(2, 2, 2) == 1.73
		
    def test_triangle_area_8(self):
        """
        Test case for the :func:`scenario1.id_71.triangle_area`.
        """
        assert triangle_area(2, 2, 3) == -1
		
    def test_triangle_area_9(self):
        """
        Test case for the :func:`scenario1.id_71.triangle_area`.
        """
        assert triangle_area(2, 3, 3) == -1
		
    def test_triangle_area_10(self):
        """
        Test case for the :func:`scenario1.id_71.triangle_area`.
        """
        assert triangle_area(3, 3, 3) == 3.89
		
		
# id_71_Test.py
from scenario1.id_71 import triangle_area

import pytest

class id_71_Test:
    """
    Test class of id_71.py.
    It contains ten unit test cases for the :func:`scenario1.id_71.triangle_area`.
    """
		
    def test_triangle_area_1(self):
        """
        Test case for the :func:`scenario1.id_71.triangle_area`.
        """
        assert triangle_area(3, 4, 5) == 6.0
		
    def test_triangle_area_2(self):
        """
        Test case for the :func:`scenario1.id_71.triangle_area`.
        """
        assert triangle_area(13, 14, 15) == 84.0
		
    def test_triangle_area_3(self):
        """
        Test case for the :func:`scenario1.id_71.triangle_area`.
        """
        assert triangle_area(1, 1, 1) == 0.43
		
    def test_triangle_area_4(self):
        """
        Test case for the :func:`scenario1.id_71.triangle_area`.
        """
        assert triangle_area(1, 2, 3) == -1
		
    def test_triangle_area_5(self):
        """
        Test case for the :func:`scenario1.id_71.triangle_area`.
        """
        assert triangle_area(1, 1, 2) == -1
		
    def test_triangle_area_6(self):
        """
        Test case for the :func:`scenario1.id_71.triangle_area`.
        """
        assert triangle_area(1, 2, 2) == -1
		
    def test_triangle_area_7(self):
        """
        Test case for the :func:`scenario1.id_71.triangle_area`.
        """
        assert triangle_area(2, 2, 2) == 1.73
		
    def test_triangle_area_8(self):
        """
        Test case for the :func:`scenario1.id_71.triangle_area`.
        """
        assert triangle_area(2, 2, 3) == -1
		
    def test_triangle_area_9(self):
        """
        Test case for the :func:`scenario1.id_71.triangle_area`.
        """
        assert triangle_area(2, 3, 3) == -1
		
    def test_triangle_area_10(self):
        """
        Test case for the :func:`scenario1.id_71.triangle_area`.
        """
        assert triangle_area(3, 3, 3) == 3.89
		
		
# id_72.py

def is_prime(n):

    if n <= 1:
        return False
    for i in range(2, n):
        if n % i == 0:
            return False
    return True

# id_72_Test.java
from scenario1.id_72 import is_prime

import pytest

class id_72_Test:
    """
    Test class of id_72.py.
    It contains ten unit test cases for the :func:`scenario1.id_72.is_prime`.
    """
		
    def test_is_prime_1(self):
        """
        Test case for the :func:`scenario1.id_72.is_prime`.
        """
        assert is_prime(1) == False
		
    def test_is_prime_2(self):
        """
        Test case for the :func:`scenario1.id_72.is_prime`.
        """
        assert is_prime(2) == True
		
    def test_is_prime_3(self):
        """
        Test case for the :func:`scenario1.id_72.is_prime`.
        """
        assert is_prime(3) == True
		
    def test_is_prime_4(self):
        """
        Test case for the :func:`scenario1.id_72.is_prime`.
        """
        assert is_prime(4) == False
		
    def test_is_prime_5(self):
        """
        Test case for the :func:`scenario1.id_72.is_prime`.
        """
        assert is_prime(5) == True
		
    def test_is_prime_6(self):
        """
        Test case for the :func:`scenario1.id_72.is_prime`.
        """
        assert is_prime(6) == False
		
    def test_is_prime_7(self):
        """
        Test case for the :func:`scenario1.id_72.is_prime`.
        """
        assert is_prime(7) == True
		
    def test_is_prime_8(self):
        """
        Test case for the :func:`scenario1.id_72.is_prime`.
        """
        assert is_prime(8) == False
		
    def test_is_prime_9(self):
        """
        Test case for the :func:`scenario1.id_72.is_prime`.
        """
        assert is_prime(9) == False
		
    def test_is_prime_10(self):
        """
        Test case for the :func:`scenario1.

File: RQ1_Test_Generation/StarCoder_Data/HumanEvalJava_output/original/id_71T2000Test.java
// TriangleArea1Test.java
package original;


import org.junit.jupiter.api.Test;
import static org.junit.jupiter.api.Assertions.*;

/**
* Test class of {@link TriangleArea1}.
* It contains ten unit test cases for the {@link TriangleArea1#triangleArea(int, int, int)} method.
*/
class TriangleArea1Test {

    /**
    * Test case for {@link TriangleArea1#triangleArea(int, int, int)} with three positive integers.
    */
    @Test
    void testTriangleArea1() {
        assertEquals(6.0, TriangleArea1.triangleArea(3, 4, 5));
        assertEquals(8.18, TriangleArea1.triangleArea(4, 8, 5));
        assertEquals(1.73, TriangleArea1.triangleArea(2, 2, 2));
        assertEquals(0.43, TriangleArea1.triangleArea(1, 1, 1));
    }

    /**
    * Test case for {@link TriangleArea1#triangleArea(int, int, int)} with three negative integers.
    */
    @Test
    void testTriangleArea2() {
        assertEquals(-1, TriangleArea1.triangleArea(-3, -4, -5));
        assertEquals(-1, TriangleArea1.triangleArea(-4, -8, -5));
        assertEquals(-1, TriangleArea1.triangleArea(-2, -2, -2));
        assertEquals(-1, TriangleArea1.triangleArea(-1, -1, -1));
    }

    /**
    * Test case for {@link TriangleArea1#triangleArea(int, int, int)} with three zeroes.
    */
    @Test
    void testTriangleArea3() {
        assertEquals(0.0, TriangleArea1.triangleArea(0, 0, 0));
        assertEquals(0.0, TriangleArea1.triangleArea(0, 0, 1));
        assertEquals(0.0, TriangleArea1.triangleArea(0, 1, 0));
        assertEquals(0.0, TriangleArea1.triangleArea(1, 0, 0));
    }

    /**
    * Test case for {@link TriangleArea1#triangleArea(int, int, int)} with three positive and negative integers.
    */
    @Test
    void testTriangleArea4() {
        assertEquals(-1, TriangleArea1.triangleArea(3, -4, 5));
        assertEquals(-1, TriangleArea1.triangleArea(-3, 4, -5));
        assertEquals(-1, TriangleArea1.triangleArea(3, -4, -5));
        assertEquals(-1, TriangleArea1.triangleArea(-3, -4, 5));
    }

    /**
    * Test case for {@link TriangleArea1#triangleArea(int, int, int)} with three positive and zero integers.
    */
    @Test
    void testTriangleArea5() {
        assertEquals(0.0, TriangleArea1.triangleArea(3, 0, 5));
        assertEquals(0.0, TriangleArea1.triangleArea(3, 5, 0));
        assertEquals(0.0, TriangleArea1.triangleArea(0, 3, 5));
        assertEquals(0.0, TriangleArea1.triangleArea(5, 3, 0));
    }

    /**
    * Test case for {@link TriangleArea1#triangleArea(int, int, int)} with three negative and zero integers.
    */
    @Test
    void testTriangleArea6() {
        assertEquals(0.0, TriangleArea1.triangleArea(-3, 0, -5));
        assertEquals(0.0, TriangleArea1.triangleArea(-3, -5, 0));
        assertEquals(0.0, TriangleArea1.triangleArea(0, -3, -5));
        assertEquals(0.0, TriangleArea1.triangleArea(-5, -3, 0));
    }

    /**
    * Test case for {@link TriangleArea1#triangleArea(int, int, int)} with three positive and one integers.
    */
    @Test
    void testTriangleArea7() {
        assertEquals(1.0, TriangleArea1.triangleArea(3, 1, 5));
        assertEquals(1.0, TriangleArea1.triangleArea(3, 5, 1));
        assertEquals(1.0, TriangleArea1.triangleArea(1, 3, 5));
        assertEquals(1.0, TriangleArea1.triangleArea(5, 3, 1));
    }

    /**
    * Test case for {@link TriangleArea1#triangleArea(int, int, int)} with three negative and one integers.
    */
    @Test
    void testTriangleArea8() {
        assertEquals(1.0, TriangleArea1.triangleArea(-3, 1, -5));
        assertEquals(1.0, TriangleArea1.triangleArea(-3, -5, 1));
        assertEquals(1.0, TriangleArea1.triangleArea(1, -3, -5));
        assertEquals(1.0, TriangleArea1.triangleArea(-5, -3, 1));
    }

    /**
    * Test case for {@link TriangleArea1#triangleArea(int, int, int)} with three positive and two integers.
    */
    @Test
    void testTriangleArea9() {
        assertEquals(2.0, TriangleArea1.triangleArea(3, 2, 5));
        assertEquals(2.0, TriangleArea1.triangleArea(3, 5, 2));
        assertEquals(2.0, TriangleArea1.triangleArea(2, 3, 5));
        assertEquals(2.0, TriangleArea1.triangleArea(5, 3, 2));
    }

    /**
    * Test case for {@link TriangleArea1#triangleArea(int, int, int)} with three negative and two integers.
    */
    @Test
    void testTriangleArea10() {
        assertEquals(2.0, TriangleArea1.triangleArea(-3, 2, -5));
        assertEquals(2.0, TriangleArea1.triangleArea(-3, -5, 2));
        assertEquals(2.0, TriangleArea1.triangleArea(2, -3, -5));
        assertEquals(2.0, TriangleArea1.triangleArea(-5, -3, 2));
    }
}
